app/tag/[slug]/page.tsx

```diff
@@ -21,6 +21,22 @@ export async function generateMetadata({
     notFound()
   }
 
+  const { totalAmount } = await fetchTagPosts({
+    take: 0,
+    skip: 0,
+    slug,
+    withAmount: true,
+  })
+
+  const robotsObj =
```

Comment:
[nit] 
1. 建議盡量跟其他人統一一個變數命名，讓功能一致的變數比較好被比較

```suggestion
  const robotsMetaData =
```

2. 建議 rrobotsObj 變數位置跟下方 assign 物件的排序位置要相同。
`
 {
    ...defaultMetadata,
    ...robotsObj,
    title,
 `

Reply:
好～